src/util/evalUnits.ts

```diff
@@ -29,8 +33,10 @@ const si = [
   { value: 1e12, symbol: 'T' },
   { value: 1e15, symbol: 'P' },
   { value: 1e18, symbol: 'E' },
-  { value: 1e21, symbol: 'Z' },
-  { value: 1e24, symbol: 'Y' }
+  // This may seem crazy but 1e21 and 1e24 were actually passed to BN like that and making it break
+  // (while 1e18 is transforming to format seen below)
+  { value: 1000000000000, symbol: 'Z' },
+  { value: 1000000000000000, symbol: 'Y' }
```

Comment:
Hmm, sorry but I'm pretty sure this is plain wrong. This has way too little zeros (that number is less than `1e18`) and should be left out as `1e21`.  I don't understand the reason for changing it in the first place.

Reply:
Yeah you are right on that. My bad. Will convert all to `new BN().pow(new BN())` as you propose [here](https://github.com/paritytech/parity-bridges-ui/pull/204#discussion_r662406151).
Makes much more sense